docs/developer-mode.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Developer Mode
+layout: default
+nav_order: 9
+---
+
+## Developer Mode
+
+
+{: .warning }
+Entering and exiting Developer mode will **wipe all** user data!
+Backup anything if you have to.
+
+{: .warning }
+"(placeholder text) insert warning for enrolled (school, buisness) chromebooks"
+
+"(placeholder text) insert detailed but breif description of developer mode"
+
+### Enabling Developer Mode
+1. Press `ESC` + `Refresh` (arrow icon) + `Power` at the same time. This should bring your system into recovery mode.
+
+2. Press `CTRL` +  `D` at the same time.
+
+3. Follow the on screen instructions, then once it reboots, you should be in developer mode.
+
+4. Setup the system like normal.
+
+{: .warning }
+**Do not enable debugging features.**
+
+### Disabling Developer Mode
```

Comment:
IMO this is like the same problem when we had both installing and flashing fw on the same page, disabling dev mode can go to a new page for detailed dev mode info.